### ISSUE TITLE
[BUG] "Muting glyphs" not working

### ISSUE THREAD

**Issue by ellawatkinsdulaney** [bokeh/bokeh#11383]
Hello! 

Here is my relevant version information:
![image](https://user-images.githubusercontent.com/85522804/123344220-43454580-d508-11eb-96fa-20ed8c02dde6.png)

I was having trouble muting glyphs on a more complicated graph, so I decided to re-create one of the bokeh examples to see if it is was my own graph or something to do with the code. I am following the simple line example, and then trying to apply the ability to mute lines. (Pictures first, followed by actual code immediately after)
![image](https://user-images.githubusercontent.com/85522804/123345167-58bb6f00-d50a-11eb-97ed-19ed66e2de6c.png)

If you use "hide", it works just fine"
![image](https://user-images.githubusercontent.com/85522804/123344381-96b79380-d508-11eb-9f91-51c0f5184a8c.png)

However if you use "mute", the legend text mutes but the line does not: 
![image](https://user-images.githubusercontent.com/85522804/123344407-acc55400-d508-11eb-8678-c04d29fbe126.png)

Thank you!

```
import bokeh
import holoviews as hv
hv.extension('bokeh')

from bokeh.plotting import figure, show
import bokeh.palettes
from bokeh.io import show

x = [1, 2, 3, 4, 5]
y = [6, 7, 2, 4, 5]

p = figure(title="Simple line example", x_axis_label='x', y_axis_label='y')
p.line(x, y, legend_label="Temp.", line_width=2)
p.line(y, x, legend_label="Tempted", line_width=2, color='green')

p.legend.click_policy="mute"
show(p)
```

**Comment by bryevdv** [comment]
Hi @ellawatkinsdulaney Please post actual text code for complete [Minimal Reproducible Examples](https://stackoverflow.com/help/minimal-reproducible-example), and avoid screenshots of code. Screenshots cannot be easily copy and pasted in order to be investigated, and also are not accessible to anyone who might be using a screen reader or other assistive tools. Please update the issue with an actual script.

Additionally, MREs here on the bokeh tracker should be *pure-Bokeh* code samples, i.e now holoviews code. Holoviews is a separate project and organization maintained by different people. We need a pure-Bokeh MRE to ascertain that a problem does actually lie within Bokeh.

**Comment by ellawatkinsdulaney** [comment]
@bryevdv Sorry about that! This is my first time raising an issue!
My example has been fixed with the real code that can be copy-pasted immediately :) 
Thanks!

**Comment by bryevdv** [comment]
@ellawatkinsdulaney I think this is a duplicate of https://github.com/bokeh/bokeh/issues/8390

TLDR, currently it is necessary to explicitly provide a muted color or muted alpha for there to be any visible effect. (Details in linked issue)
